### PR TITLE
Highlight search terms on search results page

### PR DESCRIPTION
**Pull request by davidmz**
It would be better if server code will also be changed to collapse comments on search result page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "babel-polyfill": "^6.0.16",
     "babel-runtime": "^5.8.25",
     "classnames": "^2.2.0",
+    "escape-string-regexp": "^1.0.5",
```
Comment (indeyets) [inline package.json]: We already use lodash. And lodash has https://lodash.com/docs/4.16.4#escapeRegExp
will it work? Are there any problems with it?

Comment (davidmz) [reply package.json]: My bad. I copied this dependency from BF (which don't use lodash). I will fix it.

**Comment by indeyets** [comment]
please rebase. the code is out of sync

**Comment by davidmz** [comment]
Rebase done.

**Pull request merged**